packages/components/organisms/f-checkout/src/demo/checkout-dinein.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "12345",
+  "serviceType": "dinein",
+  "customer": {
+    "firstName": "Joe",
+    "lastName": "Bloggs",
+    "phoneNumber": "+447111111111",
+    "dateOfBirth": "1980-01-30",
+    "emailAddress": "joe.bloggs@justeattakeaway.com"
+  },
+  "fulfilment": {
+    "time": {
+      "from": "2020-01-01T00:00+00:00",
+      "to": "2020-01-01T00:00+00:00"
+    },
+    "table": {
+      "identifier": "10"
```

Comment:
QUESTIONS:

1. Do we validate that a table entered exists?
2. Do we have a list of ones that exist for example?
3. Is there any reason why it wouldn't be a number?

Reply:
It's all at an MVP stage right now. I believe in the long run we will want the restaurant to provide us with a list of table numbers/names but until then, it's a text input limited to 12 characters. 

And unfortunately some restaurants name their tables 🤢